components/cam/src/physics/cam/flux_avg.F90

```diff
@@ -206,12 +210,12 @@ subroutine smooth(new, old, res, nstep, deltat, ncol)
    ! but if it is large we will add it at the rate required to put
    ! the residual back into the flux over a 2 hour period
```

Comment:
Note that the default is a 4-hour period now, not a 2-hour period.

Reply:
correct, but this does not affect non-MMF E3SM because it is only used by the MMF.

Reply:
Right I mean the comment here is incorrect now.

Reply:
oh, sorry, misunderstood, will fix.